src/main/java/id/ac/itb/logistik/ditlog/model/User.java

```diff
@@ -5,68 +5,97 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.persistence.Table;
+import java.sql.Date;
 
 @Entity
 @Table(name = "SILOG_USER")
 public class User {
-
   @Id
   @GeneratedValue
   @Column(name = "ID_USER")
-  Long idUser;
+  private Long idUser;
 
   @Column(name = "USER_NAME")
-  String username;
+  private String username;
 
   @Column(name = "USER_PASSWORD")
-  String password;
-
-  @Column(name = "ID_EMPLOYEES")
-  Long idEmployee;
+  private String password;
 
   @Column(name = "EMAIL")
-  String email;
+  private String email;
+
+  @Column(name = "ID_RESPONSIBILITY")
+  private Long idResponsibility;
+
+  @Column(name = "VENDOR_ID")
+  private Long vendorId;
 
```

Comment:
please complete user attribute for mapping. you can see reference on branch hotfix